### DISCUSSION TITLE
Not actually one second?

### DISCUSSION THREAD

**Discussion by simonleeADL** [KyleKun/one_second_diary#119] [General]
Hi! My one-second clips in the app seem to be slightly longer than one second? They're at 30fps, so nothing weird there. The screenshot shows VLC considering the 1 second mark to be ~2/3 the way through.
![Screenshot_20240103_000318_VLC](https://github.com/KyleKun/one_second_diary/assets/48544011/09a29dbb-72a8-4505-818b-60220b2415fc)

**Comment by KyleKun** [comment]
Hi! That's right, the app adds some extra milliseconds by default because 1 second literal is too short, the output movie looks way better with this addition.

**Reply by simonleeADL** [reply]
Oh, interesting. Is there a way to force it to be one second exactly? I like the rhythm to it, and keeps the full-year video to roughly 6 minutes.

**Reply by KyleKun** [reply]
Makes sense, I'll add this option to the preferences page, it's simple to implement so it will be available in the next update already (in a couple days).

**Reply by simonleeADL** [reply]
Wondeful, thankyou!

**Reply by simonleeADL** [reply]
Hi! Any progress on the feature? I'm very keen to use the app!

**Reply by clinden** [reply]
Hi @KyleKun, I followed your idea an created PR #140. Hope it helps 😃

**Reply by KyleKun** [reply]
Thanks @clinden !! I will review it soon.
